.gitignore

```diff
@@ -15,5 +15,9 @@ node_modules
 **.shadow-cljs
 package-lock.json
 public/
-.clj-kondo/.cache
-.clerk
\ No newline at end of file
+/.clj-kondo/
```

Comment:
the other stuff in this directory gets copied in by clj-kondo and gets used to lint macros etc in dependencies; are you thinking that each user should run that command on their own?

Reply:
What happened was I started using Calva (Clojure plugin for VS Code) and found the following uncommitted files, from git's perspective:
```bash
	.calva/
	.clj-kondo/babashka/
	.clj-kondo/org.mentat/
	.clj-kondo/rewrite-clj/
	.clj-kondo/taoensso/
	.cpcache/
	.lsp/
```
so I wanted to preserve the status quo unless we figure out what the value of checking in those things would be. I am guided by [this discussion](https://clojurians-log.clojureverse.org/clj-kondo/2022-03-18), which, while equivocal (as I read it), seems to indicate that at least I'm doing no harm